ldfparser/parser.py

```diff
@@ -148,9 +148,17 @@ def _populate_ldf_sporadic_frames(json: dict, ldf: LDF):
 
 def _populate_ldf_nodes(json: dict, ldf: LDF):
     nodes = _require_key(json, 'nodes', 'Missing Nodes section.')
+
     if nodes.get('master'):
         master_node = nodes['master']
-        ldf._master = LinMaster(master_node['name'], master_node['timebase'], master_node['jitter'])
+        is_j2602_protocol = ldf.get_protocol_version().use_j2602
+        default_max_header_length = 48 if is_j2602_protocol else None
+        default_response_tolerance = 0.4 if is_j2602_protocol else None
+        if master_node['max_header_length'] is None:
+            master_node['max_header_length'] = default_max_header_length
+        if master_node['response_tolerance'] is None:
+            master_node['response_tolerance'] = default_response_tolerance
+        ldf._master = LinMaster(**master_node)
```

Comment:
At the moment these two new properties won't hurt anyone, the logic here could be simplified to setting defaults if not already set.

Later a lot of these different node variants will have to be split in their parsing.